### PR TITLE
[ fix ] some mistakes

### PR DESCRIPTION
**Pull request by spcfox**
This should be an example of an implicit argument
https://github.com/stefan-hoeck/idris2-tutorial/blob/1c89da79deaa061598e82ab56f17aa3af3e9d15c/src/Tutorial/Eq.md?plain=1#L486-L499

### CODE REVIEW

**Comment by spcfox** [comment]
Did you mean «try it»?
https://github.com/stefan-hoeck/idris2-tutorial/blob/1c89da79deaa061598e82ab56f17aa3af3e9d15c/src/Tutorial/Eq.md?plain=1#L897

File: src/Tutorial/Prim.md [hunk 1]
```diff
@@ -542,10 +542,9 @@ integer types (`Int8`, `Int16`, `Int32`, and `Int64`) come with
...
-modulo `2^bitsize` and adding the lower bound (a negative number)
-if the result is still out of range. For instance, for `Int8`, all operations
-calculate their results modulo 256, subtracting 128 if the
-result is still out of bounds:
+modulo `2^bitsize` and subtracting `2^bitsize` if the result is still out of
+range. For instance, for `Int8`, all operations calculate their results modulo
+256, subtracting 256 if the result is still out of bounds:
 
 ```repl
 Main> the Int8 2 * 127
```
Comment (spcfox) [inline src/Tutorial/Prim.md:550]: Following the old explanation:
`2 * 127 = 254 % 256 - 128 = 126`

The correct way to subtract `2^bitsize`:
`2 * 127 = 254 % 256 - 256 = -2`

**Comment by spcfox** [comment]
https://github.com/stefan-hoeck/idris2-tutorial/blob/1c89da79deaa061598e82ab56f17aa3af3e9d15c/src/Tutorial/Prim.md?plain=1#L919-L934

Maybe it would be better to write
```idris
namespace Escaped
  export
  fromString : (s : String) -> Escaped
  fromString s = MkEscaped (escape s) s Refl
```
for automatic escaping during compilation? Currently the following code causes a compile time error:
```idris
escaped : Escaped
escaped = "<Hello World!>"
```

**Comment by spcfox** [comment]
Or it could be an implementation of the FromString
```idris
implementation FromString Escaped where
  fromString s = MkEscaped (escape s) s Refl
```

**Comment by stefan-hoeck** [comment]
> Or it could be an implementation of the FromString:
> 
> ```idris
> implementation FromString Escaped where
>   fromString s = MkEscaped (escape s) s Refl
> ```
> 
> I'll add this to my PR if you agree

Let's discuss this in its own PR. The two versions - your and mine - have distinct performance characteristics. In the current version of `fromString`, there will be no call to `escape` with strings where this is not required. On the other hand, `fromString` will not work with all string literals.

**Comment by stefan-hoeck** [comment]
Thanks a lot for these corrections.

**Pull request merged**